internal/output/plain.go

```diff
@@ -105,8 +105,14 @@ func wordWrap(text string) string {
 		result = append(result, crop.Line)
 	}
 	suffix := ""
-	if strings.HasSuffix(text, "\n") {
-		suffix = "\n"
+
+	// Check for line endings at the end of the string
+	// We don't care about runes here since we are only looking at line endings
+	for x := len(text) - 1; x >= 0; x-- {
+		if string(text[x]) != "\n" {
+			break
+		}
+		suffix += "\n"
```

Comment:
Another verification; Should this treatment exist for each line?

Reply:
I don't understand the question?

If the input had 5 line endings at the end of the string then the output should have 5 line endings at the end of the string.